manifests/base/argo-server/argo-server-deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       serviceAccountName: argo-server
       containers:
         - name: argo-server
-          image: quay.io/argoproj/argocli:latest
+          image: greenopsinc/argo-server:3.3.8
```

Comment:
The server is started via the CLI, so make sure that as a part of the containerization steps that it is the CLI image that is run (there should probably be info on this in the Argo Workflows repo)

Reply:
We're already using cli build for that, so no issues with the current approach. I'll update our repo to the latest stable version and create new docker image.